src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -152,6 +153,18 @@ public Task FavIcon()
         [RavenAction("/auth-error.html", "GET", AuthorizationStatus.UnauthenticatedClients)]
         public Task StudioAuthError()
         {
+            var feature = HttpContext.Features.Get<IHttpAuthenticationFeature>() as RavenServer.AuthenticateConnection;
+            var authStatus = feature?.Status;
+            
+            if (authStatus is RavenServer.AuthenticationStatus.ClusterAdmin or
```

Comment:
do we have some method to call and check if we are authorized to avoid listing AuthenticationStatus here?

Reply:
don't think so, should create such method ? @grisha-kotler

Reply:
or let's reverse condition.